### DISCUSSION TITLE
Vendor file not loading

### DISCUSSION THREAD

**Discussion by codingwithrk** [php-flasher/php-flasher#186] [General]
When i use `wire:naviage` the below vender files are not loading when I refresh page they are loading

`{
  "/vendor/flasher/flasher.min.js": "/public/vendor/flasher/flasher.min.js?id=f710e711ba04506eb4d82fa27cd5ec3d",
  "/vendor/flasher/flasher.min.css": "/public/vendor/flasher/flasher.min.css?id=edd6a85a799ab17530dea278ab8f4337",
  "/vendor/flasher/flasher-notyf.min.js": "/public/vendor/flasher/flasher-notyf.min.js?id=d8778adb84f0e2521d413c9ca995309c",
  "/vendor/flasher/flasher-notyf.min.css": "/public/vendor/flasher/flasher-notyf.min.css?id=53f57ecf59a7045f8029243bde3ed054"
}`

**Comment by yoeunes** [comment]
Hi @CodingwithRK,

Thank you for bringing this to issue to my attention. could you please provide a GitHub repository where this behavior can be replicated? As I don't regularly work with Laravel and Livewire, having a reproducible example would greatly assist in diagnosing and resolving the problem efficiently.

Best regards,
Younes

**Reply by codingwithrk** [reply]
It is working fine in local but in VPS server it not working.
